apps/frontend/src/app/api/zapier/measurement-fields/route.ts

```diff
@@ -0,0 +1,134 @@
+import { type NextRequest } from "next/server";
+import { ErrorWithCode, zapierUserAuthMiddleware } from "../middleware";
+import { api } from "~/trpc/server";
+
+// see https://zapier.github.io/zapier-platform-schema/build/schema.html#fieldschema
+type Field = {
+  key: string;
+  label?: string;
+  helpText?: string;
+  type?:
+    | "string"
+    | "text"
+    | "integer"
+    | "number"
+    | "boolean"
+    | "datetime"
+    | "file"
+    | "password"
+    | "copy";
+  required?: boolean;
+  placeholder?: string;
+  default?: string;
+  dynamic?: string;
+  search?: string;
+  //see https://zapier.github.io/zapier-platform-schema/build/schema.html#fieldchoicewithlabelschema
+  choices?: Array<{
+    label: string;
+    value: string;
+    sample: string;
+  }>;
+  inputFormat?: string;
+  list?: boolean;
+  children?: Field[];
+  dict?: boolean;
+  computed?: boolean;
+  altersDynamicFields?: boolean;
+};
+
+type IResponse = Field[];
```

Comment:
I don't see the point of this type, it's only used once and it doesn't alias anything super long, just use `Field[]` in the type annotation on line 84